lib/fixed-points.nix

```diff
@@ -77,6 +77,12 @@ rec {
           super' = super // fApplied;
       in fApplied // g self super';
 
+  # Compose several extending functions of the type expected by 'extends' into
+  # one where changes made in preceding functions are made available to
+  # subsequent ones.
+  composeExtensionss =
+    lib.foldr (x: y: composeExtensions x y) (self: super: {});
+
```

Comment:
A few things:

1. I'm not a huge fan of this naming.  It almost looks like a spell miss.  However, I don't really have a suggestion for a better name.
2. Could you add either some sort of example to the docs, or maybe a type signature?  It is hard to tell how it should be used just from the comment on the function.
3. I'd suggestion pulling this out into a separate commit, since you wouldn't want it to get removed if the hoogle thing was reverted for whatever reason.

Reply:
Done done done